src/Files.App/DataModels/RemoteDataObject.cs

```diff
@@ -0,0 +1,300 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
+using System.Runtime.Versioning;
+using System.Windows.Forms;
+using Vanara.PInvoke;
+using STATSTG = System.Runtime.InteropServices.ComTypes.STATSTG;
+
+namespace Files.App.DataModels
+{
+    // Class taken from Rx-Explorer (https://github.com/zhuxb711/RX-Explorer)
```

Comment:
We are we taking someone else's code?

Reply:
I copied the file from the one in Files.FullTrust so I wasn't the one who wrote this comment.